src/schemabuilder.rs

```diff
@@ -196,6 +196,48 @@ impl SchemaBuilder {
         Ok(self.clone())
     }
 
+    /// Add a new boolean field to the schema.
+    ///
+    /// Args:
+    ///     name (str): The name of the field.
+    ///     stored (bool, optional): If true sets the field as stored, the
+    ///         content of the field can be later restored from a Searcher.
+    ///         Defaults to False.
+    ///     indexed (bool, optional): If true sets the field to be indexed.
+    ///     fast (str, optional): Set the bool options as a single-valued fast
```

Comment:
This documentation does not look right for `fast: bool` and also for how this works since the bump to Tantivy 0.20.x. But admittedly this issues seem to apply to the other schema methods as well.

Reply:
I updated the documentation to reflect the 0.20.x. There were some other typos there that I just fixed up.

Reply:
I know this is not part of this change, but while we are discussing this:

* I think the sentence about "array access" is not correct any more because fast fields use a more complex encoding since 0.20.x which is slightly slower but much more flexible.
* IIRC, this flexibility primarily means that the cardinality limits are gone, i.e. all fast field can be optional and/or multi-valued and this like the best codec is determined automatically.

I think the best upstream documentation is currently https://docs.rs/tantivy/latest/tantivy/fastfield/index.html which we should probably reproduce and/or link here.

Reply:
I still have `Read access performance is comparable to that of an array lookup.` since that's still on the 0.20.x documentation you linked.

https://docs.rs/tantivy/latest/tantivy/schema/struct.NumericOptions.html#method.set_fast seems to still have this language - is that out of date?